src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -299,11 +298,27 @@ private void handleHydratePrevCommand()
 		final int hours = Math.toIntExact(timeSinceLastBreak.toHours());
 		final int minutes = Math.toIntExact(timeSinceLastBreak.toMinutes() % 60);
 		final int seconds = Math.toIntExact((timeSinceLastBreak.toMillis() / 1000) % 60);
-		final String timeString = String.format("%s hours %s minutes %s seconds since the last hydrate break",
-				hours, minutes, seconds);
+		final String timeString = timeDisplay(hours, minutes, seconds);
 		sendHydrateEmojiChatMessage(ChatMessageType.GAMEMESSAGE, timeString);
 	}
 
+	/**
+	 * <p>Handle the String formatting of the specified time.</p>
+	 * @param hours
+	 * @param minutes
+	 * @param seconds
+	 * @return Time in a string format
+	 * @since 1.1.1
+	 */
+	String timeDisplay(int hours, int minutes, int seconds) {
+		String hoursRepresentation = hours != 1 ? hours + " hours" : hours + " hour";
+		String minutesRepresentation = minutes != 1 ? minutes + " minutes" : minutes + " minute";
+		String secondesRepresentation = seconds != 1 ? seconds + " secondes" : seconds + " second";
```

Comment:
Can you correct the spelling of `secondes` to `seconds`?
Looks like the unit tests will have to be updated as well.

Reply:
Ok i'll do that.